src/parsing/parsers_declarations.cpp

```diff
@@ -2410,6 +2410,11 @@ void history_war_goal::casus_belli(association_type, std::string_view value, err
 	}
 }
 
+void history_war_goal::state_province_id(association_type, int32_t value, error_handler& err, int32_t line, war_history_context& context) {
+	auto amid = context.outer_context.state.world.province_get_abstract_state_membership_as_province(dcon::province_id(dcon::province::value_base_t(value)));
```

Comment:
This will not work. Province ids in the files do not convert 1 to 1 to internal province ids